src/mlpack/methods/ann/layer/celu_impl.hpp

```diff
@@ -29,11 +30,55 @@ CELUType<InputType, OutputType>::CELUType(const double alpha) :
   }
 }
 
-template<typename InputType, typename OutputType>
-void CELUType<InputType, OutputType>::Forward(
-    const InputType& input, OutputType& output)
+template<typename MatType>
+CELUType<MatType>::CELUType(const CELUType& other) :
+    Layer<MatType>(other),
+    alpha(other.alpha)
 {
-  output = arma::ones<OutputType>(arma::size(input));
+    // Nothing to do.
```

Comment:
Indentation should be two spaces; could you go through the PR and fix that please? :)

Reply:
Yes, sorry about that!